src/Framework/Framework/ViewModel/Validation/DotvvmRequestContextValidationExtensions.cs

```diff
@@ -19,18 +23,32 @@ public static ViewModelValidationError AddModelError(this IDotvvmRequestContext
             return error;
         }
 
-        public static ViewModelValidationError AddModelError(this IDotvvmRequestContext context, string propertyPath, string message)
+        /// <summary>
+        /// Adds a new raw validation error. This method is intended only for advanced use-cases and it partially bypasses the validation framework. 
+        /// Users of this method must provide an absolute validation path from the root viewmodel. Individual path segments need to be delimited using the '/' character. 
+        /// Example 1) /Customer/Id. Example 2) /Items/0/Price.
+        /// </summary>
+        /// <param name="absolutePath">Absolute validation path from the root viewmodel</param>
+        /// <param name="message">Validation error message</param>
+        public static ViewModelValidationError AddRawModelError(this IDotvvmRequestContext context, string absolutePath, string message)
```

Comment:
I think we should keep this overload, but make it obsolete. In many cases, it will still work, because most people were having most of the validated properties in the root view model. Moreover, we can explain in the obsolete comment what should they use now, this way they will just get an "overload not found" error.